duckari/duck.py

```diff
@@ -25,13 +26,25 @@ def decorate(func: _ListenerT) -> _ListenerT:
 
         return decorate
 
+    def command(self, name: str = None): # TODO: Add return type
```

Comment:
Why can name be None?

Reply:
I meant to do `name = name or callback.__name__`, will fix.